api/src/migrations/convert-translation-messages.js

```diff
@@ -0,0 +1,82 @@
+const _ = require('underscore'),
+  { promisify } = require('util'),
+  db = require('../db-nano'),
```

Comment:
We're trying to migration from nano to PouchDB - can you convert this migration? If possible use the Promise APIs in PouchDB to help migrate off callbacks too!